benchmarks.cc

```diff
@@ -149,6 +172,7 @@ void throughput_consumer_impl(unsigned N, Queue* queue, sum_t* consumer_sum, std
     unsigned const stop = N + 1;
     sum_t sum = 0;
 
+    typename Reclaimer::region_guard guard{};
```

Comment:
This guard should be queue-specific and for other queue types it should resolve to an empty structure.